docs/tutorials/react-native.mdx

```diff
@@ -0,0 +1,708 @@
+starting the camera you can provide multiple different settings such as
+resolution, quality, and metadata. In this example though we'll simply turn it
+on to stream the camera to the dashboard with default settings:
+
+```tsx title="/screens/Connect.tsx"
+import {
+  useJellyfishClient,
+  // highlight-next-line
+  useCamera,
+} from "@jellyfish-dev/react-native-client-sdk";
+
+// ...
+
+function ConnectScreen({ navigation }: ConnectScreenProps): JSX.Element {
+  // ...
+
+  // highlight-next-line
+  const { startCamera } = useCamera();
+
+  const connectToRoom = async () => {
+    try {
+      await connect(JELLYFISH_URL, peerToken.trim());
+
+      if (!(await grantedCameraPermissions())) {
+        return;
+      }
+
+      // highlight-next-line
+      await startCamera();
+    } catch (e) {
+      console.log("Error while connecting", e);
+    }
+  };
+
+  // ...
+}
+
+// ...
+```
+
+### Joining the room
+
+The last step of connecting to the room would be actually joining the room - so
+that your camera track is visible to other users. To do this simply use the `join` function
+from the `useJellyfishClient` hook.
+
+You can also provide some user metadata when joining. You can put there whatever
+you want, it depends on your business logic. In our example, we provide a user name.
+
+After joining the room we navigate to the next screen: Room screen.
+
+```tsx title="/screens/Connect.tsx"
+// ...
+
+function ConnectScreen({ navigation }: ConnectScreenProps): JSX.Element {
+  // highlight-next-line
+  const { connect, join } = useJellyfishClient();
+
+  const connectToRoom = async () => {
+    try {
+      await connect(JELLYFISH_URL, peerToken.trim());
+
+      if (!(await grantedCameraPermissions())) {
+        return;
+      }
+
+      await startCamera();
+
+      // highlight-next-line
+      await join({ name: "Mobile RN Client" });
+      // highlight-next-line
+      navigation.navigate("Room");
+    } catch (e) {
+      console.log("Error while connecting", e);
+    }
+  };
+
+  // ...
+}
+
+// ...
+```
+
+Now the app is ready for the first test. If everything went well you should see
+a video from your camera in the front-end dashboard. Now onto the second part:
+displaying the streams from other participants.
+
+## Displaying streams from other participants
+
+### Displaying video tracks
+
+The Room screen displays the video from the camera on your device and also
+videos from other participants as well. It also allows the user to exit the call.
+
+To get information about all participants (also the local one) in the room use
+`usePeers()` hook from Jellyfish Client. The hook returns all the participants
+with their ids, tracks and metadata. When a new participant joins or any
+participant leaves or anything else changes, the hook updates with the new
+information.
+
+To display video tracks Jellyfish Client has a dedicated component for
+displaying a video track: `<VideoRenderer>`. It takes a track id as a prop (it
```

Comment:
Jellyfish Client comes with a dedicated component for displaying a video track: `<VideoRenderer>`. 
It takes a track id as a prop (...) and, as any other `<View>` in react, a style.
`style` property gives a lot of possibilities  ... You can even animate your track!

or sth like this?